test/SAPHanaSRTools.pm

```diff
@@ -1,12 +1,12 @@
 #
 #
 # SAPHanaSRTools.pm
-# (c) 2014 SUSE Linux Products GmbH
-# (c) 2015-2016 SUSE Linux GmbH
-# (c) 2017-2022 SUSE LLC
+# Copyright:    (c) 2014 SUSE Linux Products GmbH
+# Copyright:    (c) 2015-2016 SUSE Linux GmbH
+# Copyright:    (c) 2017-2020 SUSE LLC
```

Comment:
We are going back in time for the Copyright :-)
Can you please fix this.

Reply:
Good that you checked that. I will send an update asap.

Reply:
Thanks